package.json

```diff
@@ -30,9 +30,14 @@
     ],
     "keybindings": [
       {
-          "command": "biggles.voice",
-          "key": "ctrl+m",
-          "mac": "cmd+m"
+        "command": "biggles.voice",
+        "key": "ctrl+m",
+        "mac": "cmd+m"
+      },
+      {
+        "command": "biggles.text",
+        "key": "ctrl+n",
+        "mac": "cmd+n"
```

Comment:
This is already a pretty common shortcut for a new file. `cmd/ctrl-h` doesn't seem to a have VSCode default. There are probably others too, I didn't do an exhaustive search!

Reply:
Or how about `cmd/ctrl+alt+m` for consistency? I think that's a more common pattern and uses up less prime real estate. The main use of this is voice I think.

Reply:
Sounds sensible!